src/TulsiGenerator/PBXTargetGenerator.swift

```diff
@@ -1499,9 +1495,24 @@ final class PBXTargetGenerator: PBXTargetGeneratorProtocol {
     } else {
       changeDirectoryAction = "cd \"\(workingDirectory)\""
     }
-    let shellScript = "set -e\n" +
+
+    let shellScript: String
+    if let templatePath = options[.BuildActionScriptTemplate].commonValue {
+      // Load the user specified template. The tempalte path is relative to the project
+      // root. Sub the following values:
+      // __CHANGE_DIRECTORY_ACTION__
+      // __BAZEL_COMMAND__
+      let scriptRoot = "\(workspaceRootURL.path)/\(templatePath)"
+      let templateString = try! String(contentsOfFile: scriptRoot)
```

Comment:
should we handle this error? Or do other occurrences of file loading like this have unsafe unwraps.